content/departments/security/knowledge-base/yubikey-faq.md

```diff
@@ -96,8 +96,27 @@ Go to Settings > Password and authentication > and select "Set as preferred meth
 
 - [Okta setup instructions](../../../departments/tech-ops/tools/Okta/main.md#adding-touch-id-as-a-phishing-resistant-mfa-option)
 
-<div style="position: relative; padding-bottom: 54.7112462006079%; height: 0;"><iframe src="https://www.loom.com/embed/5a78ffa19e604aeb8997bbb42d581ebd" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+<!-- <div style="position: relative; padding-bottom: 54.7112462006079%; height: 0;"><iframe src="https://www.loom.com/embed/5a78ffa19e604aeb8997bbb42d581ebd" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div> -->
+
+### How do I set up Yubikeys with Google
+
+[Google Account Security](https://myaccount.google.com/two-step-verification/security-keys)
+
+- Click the button to add a security key
```

Comment:
I would add
- Follow the link  (account security link here) to start the setup process